images/nix/default.nix

```diff
@@ -30,6 +31,7 @@ let
       gzip
       openssh
       xz
+      pkgs.path
```

Comment:
Is it not automatically part of the image, when you reference the file in NIX_PATH below?

Reply:
I just checked. For example for `nixpkgs/nix:nixos-20.03`
```
bash-4.4# echo $NIX_PATH
nixpkgs=/nix/store/k47by8lnwb37cww8qnblk4djhjqkxxr4-nixexprs.tar.xz
bash-4.4# ls /nix/store/k47by8lnwb37cww8qnblk4djhjqkxxr4-nixexprs.tar.xz
ls: cannot access '/nix/store/k47by8lnwb37cww8qnblk4djhjqkxxr4-nixexprs.tar.xz': No such file or directory
```

Reply:
fair enough.